src/components/backend-ai-session-launcher.ts

```diff
@@ -2824,6 +2962,25 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
                            helper="${_t('maxLength.64chars')}"
                            validationMessage="${_t('session.launcher.SessionNameAllowCondition')}">
             </mwc-textfield>
+            <wl-expansion class="editor" name="editor" ?disabled="${this.sessionType === 'interactive'}">
+              <span slot="title">${_t('session.launcher.BatchModeConfig')}</span>
```

Comment:
How about this expansion to be open when batch mode?
I got to know the startup command is required after making a session. Therefore, it would be nice to inform the user more explicitly that startup commands are required in batch mode.